src/workspace/workspace-stats.controller.ts

```diff
@@ -61,4 +64,24 @@ export class WorkspaceStatsController {
   ): Promise<DbWorkspaceRossIndex> {
     return this.workspaceStatsService.findRossByWorkspaceIdForUserId(workspaceStatsOptionsDto, id, userId);
   }
+
+  @Get("/most-active-contributor")
```

Comment:
most-active-contributors?

```suggestion
  @Get("/most-active-contributors")
```